builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -353,7 +353,8 @@ function store.get_formspec(dlgdata)
 			"container[0.375,0.375]",
 			"field[0,0;7.225,0.8;search_string;;", core.formspec_escape(search_string), "]",
 			"field_close_on_enter[search_string;false]",
-			"button[7.225,0;2,0.8;search;", fgettext("Search"), "]",
+            "image_button[7.3,0;0.8,0.8;", core.formspec_escape(defaulttexturedir .. "clear.png"), ";clear;]",
```

Comment:
indentation is wrong here and in many other places

Reply:
I have fixed all those, but I don't understand how they appeared here. In my original version it looked like properly.

Reply:
> but I don't understand how they appeared here. In my original version it looked like properly.

Of course you don’t. Your editor does. It uses spaces instead of tabs (and there are still some after your fixes). That’s a silly but very popular misfeature. Always disable it when working on Minetest code.